lib/mongoid/criteria/queryable/smash.rb

```diff
@@ -74,8 +74,12 @@ def [](key)
         # @return [ String ] The normalized key.
         #
         # @since 1.0.0
-        def normalized_key(name, serializer)
-          serializer && serializer.localized? ? "#{name}.#{::I18n.locale}" : name
+        def normalized_key(name, serializer, opts = {})
+          if serializer && serializer.localized? && opts[:localize] != false
```

Comment:
...sigh. This goes back to my previous statement that localize itself is a misnomer. It should really be called "only_current_locale: true" or something like that, because without it the field **will still be localized**--in fact more so because it can use fallbacks.